lib/matplotlib/tests/test_collections.py

```diff
@@ -1129,3 +1130,22 @@ def test_set_offset_units():
     off0 = sc.get_offsets()
     sc.set_offsets(list(zip(y, d)))
     np.testing.assert_allclose(off0, sc.get_offsets())
+
+
+@image_comparison(baseline_images=["test_check_masked_offsets"],
+                  remove_text=True, style="mpl20")
```

Comment:
```suggestion
                  extensions=["png"], remove_text=True, style="mpl20")
```
I agree with @oscargus that we probably only need png format for the test.  I think the other formats are only necessary if you think the behaviour would be different across formats.

Reply:
And now I found the documentation that says limiting to png is preferred: https://matplotlib.org/devdocs/api/testing_api.html#matplotlib.testing.decorators.image_comparison

Reply:
Got it. I'll rebase.